contracts/test/dummy_tokens/DummyToken.sol

```diff
@@ -14,14 +14,22 @@ contract DummyToken is MintableToken {
         string _name,
         string _symbol,
         uint _decimals,
-        uint _totalSupply)
+        uint _totalSupply,
+        address[] _initialBalanceHolders)
         public
     {
         name = _name;
         symbol = _symbol;
         decimals = _decimals;
         totalSupply_ = _totalSupply;
-        balances[msg.sender] = _totalSupply;
+
+        uint numInitialBalanceHolders = _initialBalanceHolders.length;
+
+        // Distribute the total supply evenly among the initial balance holders
```

Comment:
i would include an example of what the distribution would look like assuming an input of what we specify in our migrations so that future devs have some reference material.

Reply:
I would argue that the code and comment are sufficiently explanatory: the total supply is evenly distributed among the balance holders.  An example looking like the following feels like overkill:

totalSupply: 1000
numInitialBalanceHolders: 10
initial balance per account: 1000 / 10 = 100

Reply:
i agree with you. i think instead you should document in the migration what we expect the balances to be.

Reply:
agreed--done!